src/cbc_sdk/enterprise_edr/auth_events.py

```diff
@@ -0,0 +1,690 @@
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2023. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Model and Query Classes for Auth Events"""
+
+from cbc_sdk.base import UnrefreshableModel, NewBaseModel, FacetQuery
+from cbc_sdk.base import Query
+from cbc_sdk.errors import ApiError, TimeoutError, InvalidObjectError
+
+import logging
+import time
+
+log = logging.getLogger(__name__)
+
+
+class AuthEvents(NewBaseModel):
```

Comment:
I wonder whether the name of the class should be singular noun.

Reply:
done